src/main/java/nl/tudelft/semgroup4/GameState.java

```diff
@@ -16,9 +16,10 @@
 import org.newdawn.slick.state.StateBasedGame;
 
 public class GameState extends BasicGameState {
-
+    
+    private GameStateController gamestateController;
```

Comment:
PMD finds this name too long. It is obvious that the controller in this GameState is the gameStateController, so in my opinion, naming it simply 'controller' should be fine.

< naming